hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotDiffManager.java

```diff
@@ -160,6 +167,8 @@ public SnapshotDiffManager(ManagedRocksDB db,
         new ArrayBlockingQueue<>(DEFAULT_THREAD_POOL_SIZE),
         new ThreadPoolExecutor.CallerRunsPolicy()
     );
+
+    this.loadJobsOnStartUp();
```

Comment:
Could it be worth it to add a delay before actually resuming SnapDiff?
Or even better, we could make it event-triggered (e.g. when OM is "ready", rather than a fixed-time wait).
If so, we can do this in a new jira.

Reply:
Added a TODO.